src/main/java/me/zeroeightsix/kami/module/modules/misc/LogoutLogger.kt

```diff
@@ -44,9 +45,10 @@ object LogoutLogger : Module() {
                 if (mc.connection!!.getPlayerInfo(name) != null) continue
                 if (print.value) MessageSendHelper.sendChatMessage("$name logged out at ${pos.asString()}")
                 if (saveToFile.value) WaypointManager.add(pos, "$name Logout Spot")
+                if (announcer.value) mc.player.sendChatMessage( "KAMI BLUE on top! EZ Log, $name")
```

Comment:
I don't understand the point of this? Line 46 does the same thing but more useful

Reply:
I think `mc.player.sendChatMessage` sends a message to the server. They were meant to do `MessageSendHelper.sendServerMessage`

Reply:
This is basically AutoEz but for logout

Reply:
Yea that's still won't fix, it's essentially useless. I'd rather someone make a plugin

Reply:
it's supposed to say to chat "ez log" when someone logs near them to be toxic

Reply:
^